src/extension/features/budget/category-solo-mode/settings.js

```diff
@@ -1,9 +1,15 @@
 module.exports = {
   name: 'CategorySoloMode',
-  type: 'checkbox',
+  type: 'select',
   default: false,
   section: 'budget',
   title: 'Category Solo Mode and Toggle All Categories',
   description:
     'Keeps only the selected category open collapsing all others and adds a button to the Budget Toolbar to open or close all master categories at once.',
+  options: [
+    { name: 'Off', value: '0' },
+    { name: 'Enable Category Solo Mode', value: 'cat-solo-mode' },
+    { name: 'Enable Toggle All Categories', value: 'cat-toggle-all' },
```

Comment:
This used to be referred to as Toggle Master Categories. For consistency and to help users understand that this is the same feature, I recommend adding the word Master back into the description.